README.md

```diff
@@ -21,23 +21,23 @@ This is what processes the commands sent from the Raspberry Pi - Master and once
 
 #### File Summary
 
-The file is structed into two different types of files. First, the main file (.ino) which is what calls all the libraries, functions, and is uploaded on the Arduino. Second, the libraries this is called in the main file and houses functions, definitions, and classes.
+The file is structed into two different types of files. First, the main file (.ino) which is what calls all the libraries, functions, and is uploaded on the Arduino. Second, the libraries this is called in the main file and houses functions and definitions.
 
 ##### MoonDawgArduino.ino
 
-This is the code that is directly uploaded to the Arduino via the Arduino IDE. This has all of the function calls, the serial communication logic, and initialization of the classes.
+This is the code that is directly uploaded to the Arduino via the Arduino IDE. This has all of the function calls, the serial communication logic, and use of functions.
```

Comment:
I'd say the main Arduino file or something instead of "code that is directly uploaded" since its all uploaded to the Arduino.